### PR TITLE
Adding example for collection operator

### PR DESCRIPTION
**Pull request by muskan3006**
References #25468

### CODE REVIEW

File: akka-docs/src/test/scala/docs/stream/operators/sink/Collection.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import scala.concurrent.{ ExecutionContextExecutor, Future }
+
+object Collection {
+  implicit val system: ActorSystem = ???
+  implicit val ec: ExecutionContextExecutor = system.dispatcher
+  def collectionExample: Future[Unit] = {
+    //#collection
+    val source = Source(1 to 100)
+    val result: Future[List[Int]] = source.runWith(Sink.collection[Int, List[Int]])
+    result.map(println)
```
Comment (johanandren) [inline akka-docs/src/test/scala/docs/stream/operators/sink/Collection.scala:19]: `Future.foreach` to only side effect. Better to take a lower number of elements, say 5, and add a comment after this showing exactly what will be printed.

**Review by johanandren — COMMENTED** [review]
Looks good, one small improvement suggestion.

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Review by raboof — APPROVED** [review]
Thanks!

**Pull request merged**